README.md

```diff
@@ -55,6 +55,7 @@ Lassie can be controlled with a config file. To enable lassie for a project you
 
 ```json5
 {
+  "$schema": "https://raw.githubusercontent.com/geprog/lassie-bot-dog/main/config/lassie.schema.json",
```

Comment:
Using Github pages here should be easy:

```yml
name: Publish JSON schema
on:
  push:
    paths:
      - 'config/lassie.schema.json'
    branches:
      - main

jobs:
  deploy:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout 🛎️
        uses: actions/checkout@v2.3.1

      - name: Remove all files except lassie.schema.json
      - run: |
         cd config
         rm -v !("lassie.schema.json") 

      - name: Deploy 🚀
        uses: JamesIves/github-pages-deploy-action@4.1.4
        with:
          branch: gh-pages
          folder: config
          clean: false
```

Reply:
@lukashass You suggested to leave the url as is to keep it properly versioned.

I think we will always use  the latest anyways and as older versions would not be supported it should be fine to only release the current master version to github pages.

Reply:
I added this workflow.